.github/workflows/build-frequent.yaml

```diff
@@ -124,11 +124,80 @@ jobs:
       gcchash: ${{ needs.init-submodules.outputs.gcchash }}
       multilib: ${{ matrix.multilib }}
 
+  compare-artifacts:
```

Comment:
I'd prefer to invoke this similar to creg (where the actual yaml is a separate file we refer to). That'll make it simpler/more clear when invoking directly from the UI.

Reply:
build-frequent now invokes summary-runner.yaml for summary and comparison